include/small/small.h

```diff
@@ -236,6 +236,15 @@ small_alloc_check(struct small_alloc *alloc)
 	return slab_cache_check(alloc->cache);
 }
 
+/**
+ * Return the size of the mempool object that is actually allocated for the
+ * requested size. Note that it can return different values for the same input,
+ * depending on the current `small_mempool->used_pool'.
+ * If the object is allocated on the large slab (by malloc), 0 is returned.
+ */
+size_t
+small_get_objsize(struct small_alloc *alloc, size_t size);
```

Comment:
You need to implement this function for the ASAN version of the small allocator as well. CC @nshy.

Reply:
Done.